content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -56,12 +56,10 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 
 ## Using multiple Ingress controllers
 
-You may deploy [any number of ingress controllers](https://git.k8s.io/ingress-nginx/docs/user-guide/multiple-ingress.md#multiple-ingress-controllers) 
-within a cluster. When you create an ingress, you should annotate each ingress with the appropriate
-[`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster) 
-to indicate which ingress controller should be used if more than one exists within your cluster.
+You may deploy any number of ingress controllers using [ingress class](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class)
+within a cluster. Note the `.metadata.name` of your ingress class resource. When you create an ingress you would need that name to specify the `ingressClassName` field on your Ingress object (refer to [IngressSpec v1 reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#ingressspec-v1-networking-k8s-io)). `ingressClassName` is a replacement of older [annotation method](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation).
```

Comment:
```suggestion
You may deploy any number of ingress controllers using [ingress class](/docs/concepts/services-networking/ingress/#ingress-class)
within a cluster. Note the `.metadata.name` of your ingress class resource. When you create an ingress you would need that name to specify the `ingressClassName` field on your Ingress object (refer to [IngressSpec v1 reference](/docs/reference/generated/kubernetes-api/v1.22/#ingressspec-v1-networking-k8s-io)). `ingressClassName` is a replacement of older [annotation method](/docs/concepts/services-networking/ingress/#deprecated-annotation).
```

Reply:
Even better: link to the _new_ API reference page: [`/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec`](https://k8s.io/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec).

I'd link to https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-ingress-class-deprecated for the deprecated annotation (we could add more text to that entry at the same time).